bigquery/bq_schema.json

```diff
@@ -0,0 +1,193 @@
+{ "BigQuery Schema":
+[
+  {
+    "mode": "REQUIRED",
+    "name": "last_updated_ts",
+    "type": "TIMESTAMP"
+  },
+  {
+    "mode": "REQUIRED",
+    "name": "last_updated",
```

Comment:
Are you intentionally leaving both last_updated fields in?

Reply:
We retain the last_updated string to observe the original data format whilst adding a new field in timestamp form for indexing purposes.